### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by Acquiescence3**
Added the information requested

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,5 @@ server/chat-plugins/scavenger*.js @xfix
 server/chat-plugins/trivia.js @xfix
 server/chat-plugins/mafia*.js @HoeenCoder @whalemer
 data/mods/ssb/ @HoeenCoder
+data/mods/mafia/ @HoeenCoder
```
Comment (scheibo) [inline CODEOWNERS]: This should be one line (like line 3 above), not two

**Comment by Marty-D** [comment]
That folder doesn't even exist. What exactly are you trying to do, Kirkburn?

**Comment by Zarel** [comment]
Yeah, I'm confused who requested this, too.

**Comment by Acquiescence3** [comment]
I thought this still needed to be updated, my apologies.

**Pull request closed without merging**